pom.xml

```diff
@@ -27,6 +27,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-security -->
```

Comment:
Remove this line and dependency version (in spring boot we don't have to specify versions of spring dependencies, framework will do it for us)